juniper_official/SRX/Security/srx-cluster-flow-session-summary.rule

```diff
@@ -0,0 +1,225 @@
+ *      color to red when active sessions for a PIC slot is greater than this value.
+ *   2) "max-sessions-threshold-value" is the threshold that causes the rule to report
+ *      an anomaly.  By default it's 5000000. This rule will set a dashboard
+ *      color to red when active sessions for a PIC slot is greater than this value.
+ *  
+ */
+iceberg {
+    topic security  {
+        rule check-flow-session-summery {
+            description "Monitors and notify flow session details";
+            synopsis "flow session summery statistics analyzer"; 
+            /*
+             * Monitors the active flow session setails  and notifies via the dashboard
+             * colors when the threshold is above {{active-sessions-threshold}}
+             * it will mark as red.
+             * The color is set to yellow when the traffic is above dt-active-sessions.
+             * Otherwise the color is set to green.
+             * 
+             * Monitors the maximimum flow session setails  and notifies via the dashboard
+             * colors when the threshold is above {{max-sessions-threshold}}
+             * it will mark as red.
+             * The color is set to yellow when the traffic is above dt-max-sessions.
+             * Otherwise the color is set to green.
+             * Use fpc-pic-id and re-name slot as key for rule.
+             */
+            keys [ fpc-pic-id re-name ];
+            sensor flow-session {
+                synopsis "flow session sensor initialization";
+                description "security flow sesssion sensor to collect data from network device";
+                iAgent {
+                    file srx-cluster-flow-session-summary.yml;
+                    table SPUSessionSummaryTable;
+                    frequency 60s;
+                }
+            }
+            /*
+             * Fields defined with variables, dynamic thresholds and constant values
+             */
+            field active-sessions {
+                sensor flow-session {
+                    path active-sessions;
+                }
+                type integer;
+                description "This field contains number of active sessions for flow pic id";
+            }
+            field active-sessions-threshold {
+                constant {
+                    value "{{active-sessions-threshold-value}}";
+                }
+                type integer;
+                description "This filed contains the active flow session threshold value";
+            }
+            field dt-active-sessions {
+                formula {
+                    dynamic-threshold {
+                        algorithm 3sigma;
+                        learning-period 7d;
+                        pattern-periodicity 1d,1h;
+                        field-name "$active-sessions";
+                    }
+                }
+                type integer;
+                description "This filed contains the active flow sessions threshold value";
```

Comment:
s/"This filed contains the active flow sessions threshold value"/"ML derived active flow sessions threshold value"/g